### ISSUE TITLE
Unable to build from source on Ubuntu 20.04

### ISSUE THREAD

**Issue by Basdorsman** [huderlem/porymap#523]
Hi there, 

I am trying to build porymap from source on my Ubuntu 20.04 machine. I had first installed 

Qt 5.15.2 (x86_64-little_endian-lp64 shared (dynamic) release build; by GCC 11.2.0) on "xcb" 
OS: Ubuntu 20.04.3 LTS [linux version 5.14.0-1057-oem]

but `make` fails when the installation fails by throwing the following error:

```
/usr/bin/ld: /home/bas/anaconda3/lib/libQt5Core.so: undefined reference to `std::__exception_ptr::exception_ptr::_M_release()@CXXABI_1.3.13'
/usr/bin/ld: /home/bas/anaconda3/lib/libQt5Widgets.so: undefined reference to `std::__throw_bad_array_new_length()@GLIBCXX_3.4.29'
/usr/bin/ld: /home/bas/anaconda3/lib/libQt5Core.so: undefined reference to `std::__exception_ptr::exception_ptr::_M_addref()@CXXABI_1.3.13'
collect2: error: ld returned 1 exit status
make: *** [Makefile:740: porymap] Error 1
```

Do you think this is related to Qt not being the exact version 5.14.2? I was unable to install that version. 
Any others recognize this error or have an idea on how I could try to troubleshoot this?

**Comment by garakmon** [comment]
No, it's not an issue with your Qt version, it seems more like a gcc lib problem to me.

**Comment by Stu-Gotz** [comment]
Hello,

I am on Linux Ubuntu 22.04 and had a similar issue. After some googling and comparing other answers, I did `sudo apt-get install build-essential libgl1-mesa-dev` and tried to make again and it worked. YMMV.
